src/main/java/Wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package Wolox.training.exceptions;
+import java.lang.RuntimeException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "Book not found")
+public class BookNotFoundException extends  RuntimeException{
+    public BookNotFoundException() {
+        super("Book not found");
```

Comment:
This String can be a constant
```suggestion
    private static final String MESSAGE = "Book not found";

    public BookNotFoundException() {
        super(MESSAGE);
```